staging/publishing/rules.yaml

```diff
@@ -629,9 +629,23 @@ rules:
       branch: master
       dir: staging/src/k8s.io/kubectl
     name: master
+    dependencies:
+      - repository: apimachinery
+        branch: master
+      - repository: api
+        branch: master
+      - repository: client-go
+        branch: master
   - source:
       branch: release-1.15
       dir: staging/src/k8s.io/kubectl
     name: release-1.15
     go: 1.12.5
 
+    dependencies:
+      - repository: apimachinery
+        branch: master
+      - repository: client-go
```

Comment:
it looks like client-go is not a dependency yet, can we remove it?

Reply:
I will fix this too (I was just copying rules from another PR--there are lots for this big move). Thanks again for your review

Reply:
I re-added this. I think with all of our code moves and re-merging this is now necessary, otherwise tests start failing.